docs/basics/ipc.md

```diff
@@ -236,7 +236,7 @@ byte | effect
 For releases since 2012.05.29, kdb+ and the C-API will compress an outgoing message if
 
 -   Uncompressed serialized data has a length greater than 2000 bytes
--   Connection is not `localhost`
+-   Connection is not `localhost` nor 127.0.0.1 (nor since 4.1t 2021.06.04, a connection that resolves to being localhost)
```

Comment:
Is this better as several lines?
```suggestion
-   Connection is not `localhost`
-   Connection is not 127.0.0.1
-   Connection does not resolve to being localhost (since 4.1t 2021.06.04)
```